lib/util.js

```diff
@@ -475,10 +485,14 @@ function stylizeWithColor(str, styleType) {
   return str;
 }
 
-function stylizeNoColor(str, styleType) {
+function stylizeNoColor(str) {
   return str;
 }
 
+function getEmptyArray() {
```

Comment:
Could this named `getEmptyFormatArray`? And add a comment about these two that they are the no-op options for later polymorphism?

Reply:
Done